views/home.html

```diff
@@ -44,6 +45,23 @@
        </div>
      </div>
 
+     <div id="mocha"></div>
+
+    <script src="https://cdn.rawgit.com/jquery/jquery/2.1.4/dist/jquery.min.js"></script>
+    <script src="https://cdn.rawgit.com/Automattic/expect.js/0.3.1/index.js"></script>
```

Comment:
expect.js is a (light) analogue to mocha/chai—if we want to use the latter this should be safe to remove.